interface/patient_file/problem_encounter.php

```diff
@@ -261,98 +259,104 @@ function doclick(pfx, id) {
+                                        echo "     <td class='align-top'>" . text($ISSUE_TYPES[($row['type'])][1]) . "</td>\n";
+                                        echo "     <td class='align-top'>" . text($row['title']) . "</td>\n";
+                                        echo "     <td class='align-top'>" . text($row['comments']) . "</td>\n";
+                                        echo "    </tr>\n";
+                                        $endjs .= "pselected[" . js_escape($rowid) . "] = '';\n";
+                                    }
+                                    ?>
+                                </table>
+                            </div>
+                        </td>
+                        <td class="text-center align-top p-0 pr-3">
+                            <div class="table-responsive">
+                                <table class="table table-sm">
+                                    <tr class='head'>
+                                        <td colspan='2' class="text-center">
+                                            <input type='radio' name='form_key' value='e' onclick='clearall()' />
+                                            <span class="font-weight-bold"><?php echo xlt('Encounters Section'); ?></span>
+                                        </td>
+                                    </tr>
+                                    <tr class='head'>
+                                        <td><?php echo xlt('Date'); ?></td>
+                                        <td><?php echo xlt('Presenting Complaint'); ?></td>
+                                    </tr>
+                                    <?php
+                                    while ($row = sqlFetchArray($eres)) {
+                                        $rowid = $row['encounter'];
+                                        echo "    <tr class='detail' id='e_" . attr($rowid) . "' onclick='doclick(\"e\", " . attr_js($rowid) . ")'>\n";
+                                        echo "     <td class='align-top'>" . text(substr($row['date'], 0, 10)) . "</td>\n";
+                                        echo "     <td class='align-top'>" . text($row['reason']) . "</td>\n";
+                                        echo "    </tr>\n";
+                                        $endjs .= "eselected[" . js_escape($rowid) . "] = '';\n";
+                                    }
+                                    ?>
+                                </table>
+                            </div>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td colspan='2' class="text-center">
+                            <div class="btn-group">
+                                <button type="button" class="btn btn-primary btn-sm btn-save" name='form_save' value='<?php echo xla('Save'); ?>' disabled>
```

Comment:
this needs have have `type="submit"` to work or else the Save button is not working in Popup->Issues